data/scripts/sol-flow-node-type-gen.py.in

```diff
@@ -258,6 +258,7 @@ data_type_to_c_map = {
     "direction-vector": "struct sol_direction_vector",
     "string": "const char *",
     "location": "struct sol_location",
+    "timestamp": "time_t",
```

Comment:
not sure `time_t` is the best option for us to store a timestamp. It lacks information such as the timezone. Also, precision is low since it's in the seconds, if someone uses it to stamp smaller deltas it will be a problem.

not sure if it's worth or not, but we need to check those.